CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 [Unreleased]
 * Added _UnityEngine.Object.FindObjectOfType_ usage detection
 * Fixed names of build-generated assets in Build Report 
+* Added AudioSettings speaker mode descriptor and fix to switch it to Mono
```

Comment:
We already had a descriptor. I would say something like "
- Added Audio Speaker Mode diagnostic fixer
- Improved Audio Speaker Mode diagnostic recommendations"